### PR TITLE
Add windows ci

### PR DESCRIPTION
**Pull request by fendor**
AppVeyor Output: https://ci.appveyor.com/project/fendor/hie-bios/builds/29652292

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+version: 1.0.{build}
+environment:
+  matrix:
+ # - GHCVER: 8.8.1 Disabled until unix-compat supports ghc 8.8.1, see https://github.com/jacobstanley/unix-compat/pull/42 for example
+  - GHCVER: 8.6.4
```
Comment (ndmitchell) [inline appveyor.yml]: Should be 8.6.5

File: hie-bios.cabal [hunk 2]
```diff
@@ -66,7 +66,7 @@ Extra-Source-Files:     ChangeLog
                         tests/projects/simple-stack/simple-stack.cabal
 
 
-tested-with: GHC==8.6.4, GHC==8.4.4, GHC==8.2.2
+tested-with: GHC==8.6.4, GHC==8.4.4, GHC==8.2.2, GHC==8.8.1
```
Comment (ndmitchell) [inline hie-bios.cabal]: Strong statement to write tested with when we don't. Again, 8.6.5 would be better

File: tests/BiosTests.hs [hunk 3]
```diff
@@ -31,21 +32,25 @@ main = do
...
+                         $  testDirectory "./tests/projects/multi-stack/app/Main.hs"
+                         >> testDirectory "./tests/projects/multi-stack/src/Lib.hs"
+           ]
       ]
 
-
+linuxExlusiveTestCases :: [TestTree]
+linuxExlusiveTestCases
+  | isWindows = []
+  | otherwise = [ testCaseSteps "simple-bios" $ testDirectory "./tests/projects/simple-bios/B.hs" ]
```
Comment (ndmitchell) [inline tests/BiosTests.hs]: I'd have done `[testCaseSteps ... | not isWindows] and inlined the whole thing. Personal preference though.

File: hie-bios.cabal (hunk 2 above)
Comment (fendor) [reply hie-bios.cabal]: This was used to generate the .travis.yaml config. Modified it to reflect the actual tested ghc versions.

**Pull request merged**